src/fmu/dataio/_model/fields.py

```diff
@@ -572,6 +583,60 @@ class FMUBase(BaseModel):
     See :class:`Model`."""
 
 
+class FMUIteration(FMUBase):
+    """
+    The ``fmu`` block contains all attributes specific to FMU. The idea is that the FMU
+    results data model can be applied to data from *other* sources - in which the
+    fmu-specific stuff may not make sense or be applicable.
+    This is a specialization of the FMU block for Iteration objects.
+    """
+
+    context: CaseContext
+    """The ``fmu.context`` block contains the FMU context in which this data object
+    was produced. See :class:`Context`.  """
+
+    iteration: Iteration
+    """The ``fmu.iteration`` block contains information about the iteration this data
+    object belongs to. See :class:`Iteration`. """
+
+    workflow: Optional[Workflow] = Field(default=None)
```

Comment:
Will this `workflow` ever be used?

Reply:
I guess it will always be made on Sumo upload, so probably not much value in this field ? 🙂

Reply:
Good point 👆 
Removed `workflow`.
Also removed the optional `fmu.ert` block. Think that is in a similar boat ⛵

Done for both `FMUIteration` and `FMURealization`

Reply:
Nice 😃